README.md

```diff
@@ -287,6 +287,27 @@ We provide a mock MHS adaptor for local development and testing.
 | MOCK_MHS_SERVER_PORT                 | 8081                      | The port on which the mock MHS Adapter will run.
 | MOCK_MHS_LOGGING_LEVEL               | INFO                      | Mock MHS logging level. One of: DEBUG, INFO, WARN, ERROR. The level DEBUG **MUST NOT** be used when handling live patient data.
 
+## How to transform arbitrary json ASR payload files
+
+This is a document on how to run tests on json files to convert them into the Xml Files for analysis
```

Comment:
Remove 'This is a document on'

Maybe something like This is a interoperability testing toolt to transform arbitrary/ad-hoc json ASR payloads and access the outputs